internal/sysfs/file_test.go

```diff
@@ -100,7 +100,7 @@ func TestWriteFdNonblock(t *testing.T) {
 	// Create a buffer (the content is not relevant)
 	buf := make([]byte, 1024)
 	// Write to the file until the pipe buffer gets filled up.
-	numWrites := 100
+	numWrites := 1111
```

Comment:
I think this change itself seems fine, but could you put the documentation comment here on why we have 1111 here.